server/routes/genesetEnrichment.ts

```diff
@@ -26,7 +26,15 @@ function init({ genomes }) {
 	return async (req: any, res: any): Promise<void> => {
 		try {
 			const results = await run_genesetEnrichment_analysis(req.query as genesetEnrichmentRequest, genomes)
-			res.send(results as genesetEnrichmentResponse)
+			if (!req.query.geneset_name) {
```

Comment:
please add some comment at the top to explain the logic. when does it run gsea, when does it render a graph

Reply:
I added more comments